presentation/src/main/java/com/sopt/presentation/calendar/screen/InfoScreen.kt

```diff
@@ -0,0 +1,233 @@
+                        placeholder = "이름을 입력해주세요.",
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .padding(top = 10.dp)
+                    )
+                    Row(
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .padding(end = 4.dp, top = 4.dp),
+                        horizontalArrangement = Arrangement.End
+                    ) {
+                        Text(
+                            text = "${uiState.appointName.length}/20",
+                            style = typography.b5Regular,
+                            color = colors.gray500
+                        )
+                    }
+                }
+            }
+
+            SubHeaderText("약속 카테고리", Modifier.padding(top = 41.dp))
+
+            Row(
+                horizontalArrangement = Arrangement.spacedBy(11.dp),
+                verticalAlignment = Alignment.CenterVertically,
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(top = 10.dp)
+            ) {
+                listOf("중요", "일정", "취미", "기타").forEach { category ->
+                    val isSelected = uiState.category == category
+                    Box(
+                        modifier = Modifier
+                            .background(Color.Transparent)
+                            .clickable { viewModel.updateCategory(category) }
+                    ) {
+                        NoostakCategoryChip2(
+                            text = category,
+                            textColor = if (isSelected) Color.White else colors.black,
+                            backgroundColor = if (isSelected) colors.black else Color.White,
+                            borderColor = if (isSelected) Color.Transparent else colors.gray200
+                        )
+                    }
+                }
+            }
+
+            SubHeaderText("소요시간", Modifier.padding(top = 41.dp))
+
+            Box(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(top = 10.dp)
+                    .background(Color.White)
+                    .height(48.dp),
+                contentAlignment = Alignment.CenterStart
+            ) {
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(horizontal = 16.dp),
+                    horizontalArrangement = Arrangement.End,
+                    verticalAlignment = Alignment.CenterVertically
+                ) {
+                    BasicTextField(
+                        value = if (uiState.time == 0) "" else uiState.time.toString(),
+                        onValueChange = { newValue ->
+                            val timeValue = newValue.toIntOrNull() ?: 0
+                            viewModel.updateTime(timeValue)
+                        },
```

Comment:
P1: 다음과 같이 숫자 키보드를 제공하면 Int로 변환할 필요가 없습니다~!

```suggestion
keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Number)
```